### PR TITLE
SLSP-25

### PR DESCRIPTION
**Pull request by willbryanta**
create: creates an index of all library items, including the list of reviews

### CODE REVIEW

File: components/LibraryIndex/LibraryIndex.jsx [hunk 1]
```diff
@@ -1,5 +1,24 @@
-function LibraryIndex() {
-	return <></>
+function LibraryIndex(props) {
+	const {libraryItems} = props
+
+	const allLibraryItems = libraryItems.map((libraryItem, index) => (
+		<ul key={index}>
```
Comment (Zarbba) [inline components/LibraryIndex/LibraryIndex.jsx:5]:
```suggestion
		<ul key={libraryItem._id}>
```

File: components/LibraryIndex/LibraryIndex.jsx [hunk 2]
```diff
@@ -1,5 +1,24 @@
...
+			<li>Name: {libraryItem.name}</li>
```
Comment (Zarbba) [inline components/LibraryIndex/LibraryIndex.jsx:6]: This should be a link that naviagets to an individual item.

File: components/LibraryIndex/LibraryIndex.jsx [hunk 3]
```diff
@@ -1,5 +1,24 @@
...
+			<li>Description: {libraryItem.description}</li>
+			<li>Publication Date: {libraryItem.publicatioDate}</li>
+			<li>Author: {libraryItem.author}</li>
+			<li>
+				Reviews: <li>Description: {libraryItem.review.description}</li>
+				<li>Rating: {libraryItem.review.rating}</li>
```
Comment (Zarbba) [inline components/LibraryIndex/LibraryIndex.jsx:12]: I don't think we'll need all this information for the index.

**Review by Zarbba — CHANGES_REQUESTED** [review]

File: components/LibraryIndex/LibraryIndex.jsx (hunk 3 above)
Comment (willbryanta) [reply components/LibraryIndex/LibraryIndex.jsx]: agreed - I got rid of this info

File: components/LibraryIndex/LibraryIndex.jsx (hunk 2 above)
Comment (willbryanta) [reply components/LibraryIndex/LibraryIndex.jsx]: These should now link to each individual library item

File: components/LibraryIndex/LibraryIndex.jsx (hunk 1 above)
Comment (willbryanta) [reply components/LibraryIndex/LibraryIndex.jsx]: good shout - I've updated this

**Review by Zarbba — APPROVED** [review]
LGTM

**Pull request merged**